pywinauto/unittests/test_application.py

```diff
@@ -852,6 +852,41 @@ def test_find_elements_re(self):
         windows = findwindows.find_elements(title_re = "Untitled - Notepad")
         self.assertTrue(len(windows) >= 1)
 
+class WaitUntilDecoratorTests(unittest.TestCase):
+    """Unit tests for the always_wait_until decorator"""
+
+    def test_always_wait_until_decorator_success(self):
+        """Test always_wait_until_decorator success"""
+        
+        @always_wait_until(30, 10)
+        def foo():
+            return True
+        self.assertTrue(foo())
+		
+    def test_always_wait_until_decorator_failure(self):
+        """Test wait_until_decorator failure"""
+        
+        @always_wait_until(20, 10)

```

Comment:
There is no point in waiting for the long timeout. It just sums up in a longer wait time for the whole CI matrix. Try to use shorter intervals.